prebid-server/endpoints/openrtb2/amp.md

```diff
@@ -11,16 +11,22 @@ This document describes the behavior of the Prebid Server AMP endpoint in detail
 For a more general reference, see the [Prebid AMP Implementation Guide
 ]({{site.baseurl}}/dev-docs/show-prebid-ads-on-amp-pages.html).
 
-## `GET /openrtb2/amp?tag_id={ID}`
+## AMP Endpoint
 
-The `tag_id` ID must reference a [Stored BidRequest]({{site.baseurl}}/prebid-server/developers/stored-requests.html).
-For a thorough description of BidRequest JSON, see the [/openrtb2/auction](./auction.html) docs.
+ `GET /openrtb2/amp?tag_id={ID}`
+
+ **Parameters**
+
+ {: .table .table-bordered .table-striped }
+| Param | Scope | Type | Description |
+| --- | --- | --- | --- |
+| tag_id | Required | `String` |  The `tag_id` ID must reference a [Stored BidRequest]({{site.baseurl}}/prebid-server/developers/stored-requests.html). For a thorough description of BidRequest JSON, see the [/openrtb2/auction](./auction.html) docs. |
```

Comment:
BidRequest should probably be two words.

Reply:
Fixed.